src/inmanta/plugins.py

```diff
@@ -381,7 +382,7 @@ def resolve_type(self, plugin: "Plugin", resolver: Namespace) -> inmanta_type.Ty
         if not isinstance(self.type_expression, str):
             if typing_inspect.is_union_type(self.type_expression) and not typing.get_args(self.type_expression):
                 # If typing.Union is not subscripted, isinstance(self.type_expression, type) evaluates to False.
-                raise TypingException(None, f"Union type must be subscripted, got {self.type_expression}")
+                raise RuntimeException(stmt=None, msg=f"Union type must be subscripted, got {self.type_expression}")
```

Comment:
I think this was changed because of something I said, but looking back at it, I wonder if it should have been changed the other way around? This is the normalization stage. I don't think we should raise `RuntimeException` here (same for a few lines down). That said, we do so in some other places as well. @wouterdb I think you have a better view on what these exceptions are intended for?

Reply:
I would prefer a subtype of typingexception